install.sh

```diff
@@ -42,80 +42,105 @@ while getopts "hbdempo" FLAG; do
         m) SCRATCH_MONGODB_INSTALL=true ;;
         p) BUILD_APP=false ;;
         o) DOCS_INSTALL=true ;;
-        *) usage ;;
+        *)
+            usage
+            exit 1
+            ;;
     esac
 done
-[ ${SHOW_HELP} = true ] && usage && exit 0
+[ "$SHOW_HELP" = true ] && usage && exit 0
 
 set -e
 NODE_VERSION=22.14.0
 OS=$(uname -s)
-ARCH=$(uname -m)
 
+MIN_MINOR=9
+MAX_MINOR=12
+
+if ! command -v python3 >/dev/null 2>&1; then
+    echo "Python 3 is not installed or not in PATH"
+    exit 1
+fi
+
+# Extract major.minor version info
+PY_VER=$(python3 -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')
```

Comment:
I'm using a pyenv virtualenv on Python 3.10 but this command gives Python 3.13, I think because the `python3` command isn't shimmed as part of the virtual env.